map_model/src/objects/traffic_signals.rs

```diff
@@ -175,6 +169,14 @@ impl ControlTrafficSignal {
 
     /// Returns true if this did anything
     pub fn convert_to_ped_scramble(&mut self) -> bool {
+        self.internal_convert_to_ped_scramble(true)
+    }
+    /// Returns true if this did anything
```

Comment:
Maybe describe what "promotion" means, or rename to the more verbose `convert_to_ped_scramble_without_promoting_yields_to_protecteds`... oof that's long, just a docstring please